instance_control/readme.md

```diff
@@ -2,6 +2,15 @@
 
 The script helps to manage upgrades of Bubuku(Kafka) without terminating EBS volume.
 
+### Environment set up
```

Comment:
The script uses zaws to login to the account which is specified in cluster config json file.